tasks/release/Jira.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace MailPoetTasks\Release;
+
+class Jira {
+
+  const CHANGELOG_FIELD_ID = 'customfield_10500';
+  const RELEASENOTE_FIELD_ID = 'customfield_10504';
+
+  const PROJECT_MAILPOET = 'MAILPOET';
+  const PROJECT_PREMIUM = 'PREMIUM';
+
+  const JIRA_DOMAIN = 'mailpoet.atlassian.net';
+
+  /** @var string */
+  private $token;
+
+  /** @var string */
+  private $user;
+
+  /** @var string */
+  private $project;
+
+  public function __construct($token, $user, $project) {
+    $this->token = $token;
+    $this->user = $user;
+    $this->project = $project;
+  }
+
+  /**
+   * @see https://developer.atlassian.com/cloud/jira/platform/rest/v3/#api-api-3-project-projectIdOrKey-versions-get
+   */
+  function getVersion($version_name = null) {
+    $versions = $this->fetchFromJira("project/$this->project/versions");
+    if($version_name === null) {
+      return end($versions);
+    }
+    foreach($versions as $version) {
+      if($version_name === $version['name']) {
+        return $version;
+      }
+    }
+    throw new \Exception('Unknown project version');
+  }
+
+  /**
+   * @see https://developer.atlassian.com/cloud/jira/platform/rest/v3/#api-api-3-search-get
+   */
+  function getIssuesDataForVersion($version) {
+    $changelog_id = self::CHANGELOG_FIELD_ID;
+    $release_note_id = self::RELEASENOTE_FIELD_ID;
+    $issues_data = $this->fetchFromJira("/search?fields=key,$changelog_id,$release_note_id,status&jql=fixVersion={$version['id']}");
+    // Sort issues by importance of change (Added -> Updated -> Improved -> Fixed -> Others)
```

Comment:
I just noticed sometimes we used also "Changed". Might be safer to add it as well.